Docs/user_guide/modules/webserver.md

```diff
@@ -0,0 +1,73 @@
+# WebServer
+
+## Overview
+
+Creates a fairly simple but pretty powerful web server that can respond to http(s) requests with data from the current Slicer or modify the Slicer state.  This module is meant to be the basis for implemeting web applications that use Slicer as a remote render / computation engine or for controlling your Slicer instance for interaction with other system code like shell scripts or other applications.
+
+There are three basic types of endpoints:
+- **Static:** Hosts files out of the module's `docroot` like any standard http server.  Currently this is used just for examples.
```

Comment:
We should separate potentially clinically useful examples from from tests and pure technology demonstrations. Clinically useful stuff should be at the top and the rest (timeimage, timecube, spinning cube) should be in a separate section, but preferably in a separate page (can be linked from the main page).